llvm/unittests/ADT/DenseMapTest.cpp

```diff
@@ -379,6 +379,17 @@ TEST(DenseMapCustomTest, EqualityComparison) {
   EXPECT_NE(M1, M3);
 }
 
+TEST(DenseMapCustomTest, InsertRange) {
+  DenseMap<int, int> M;
+
+  std::pair<int, int> InputVals[3] = {{0, 0}, {0, 1}, {1, 2}};
+  M.insert_range(InputVals);
+
+  EXPECT_EQ(2u, M.size());
```

Comment:
```suggestion
  EXPECT_EQ(M.size(), 2u);
```
We typically put constants on the RHS of comparisons, especially with how gtest presents actual/expected values.

Reply:
Done